options/locale/locale_en-US.ini

```diff
@@ -1989,6 +1990,13 @@ contributors.loading_title_failed = Could not load contributions
 contributors.loading_info = This might take a bit…
 contributors.component_failed_to_load = An unexpected error happened.
 
+
+code_frequency = Code Frequency
+code_frequency.loading_title = Loading code frequency...
+code_frequency.loading_title_failed = Could not load code frequency
+code_frequency.loading_info = This might take a bit…
+code_frequency.component_failed_to_load = An unexpected error happened.
```

Comment:
Please never ever duplicate translations that already exist! You can rename the ones made for commit graph and use them here. Also, use placeholders if the only thing that changes is the type of data being loaded ("Could not load %s")

Reply:
I need help here: https://github.com/go-gitea/gitea/pull/29210#discussion_r1493414014